org-formation/075-inspector/inspector.yaml

```diff
@@ -0,0 +1,91 @@
+AWSTemplateFormatVersion: '2010-09-09-OC'
+OrganizationBindings:
+  InspectorBinding:
+Parameters:
+  ScheduleExpression:
+    Type: String
+    Default: "rate(7 days)"
+  ScheduleState:
+    Type: String
+    Default: "ENABLED"
+    AllowedValues:
+      - "ENABLED"
+      - "DISABLED"
+  AssessmentDurationInSeconds:
+    Type: Number
+    Default: 3600
+    MinValue: 180
+    MaxValue: 86400
+Mappings:
+  RulesPackagesAmazonInspectorArns: # ARNs derived from https://docs.aws.amazon.com/inspector/latest/userguide/inspector_rules-arns.html
+    us-east-1:
+      CommonVulnerabilitiesExposures: arn:aws:inspector:us-east-1:316112463485:rulespackage/0-gEjTy7T7
+      CISOperatingSystemBenchmarks: arn:aws:inspector:us-east-1:316112463485:rulespackage/0-rExsr2X8
+      NetworkReachability: arn:aws:inspector:us-east-1:316112463485:rulespackage/0-PmNV0Tcd
+      SecurityBestPractices: arn:aws:inspector:us-east-1:316112463485:rulespackage/0-R01qwB5Q
+    us-east-2:
+      CommonVulnerabilitiesExposures: arn:aws:inspector:us-east-2:646659390643:rulespackage/0-JnA8Zp85
+      CISOperatingSystemBenchmarks: arn:aws:inspector:us-east-2:646659390643:rulespackage/0-m8r61nnh
+      NetworkReachability: arn:aws:inspector:us-east-2:646659390643:rulespackage/0-cE4kTR30
+      SecurityBestPractices: arn:aws:inspector:us-east-2:646659390643:rulespackage/0-AxKmMHPX
+    us-west-1:
+      CommonVulnerabilitiesExposures: arn:aws:inspector:us-west-1:166987590008:rulespackage/0-TKgzoVOa
+      CISOperatingSystemBenchmarks: arn:aws:inspector:us-west-1:166987590008:rulespackage/0-xUY8iRqX
+      NetworkReachability: arn:aws:inspector:us-west-1:166987590008:rulespackage/0-TxmXimXF
+      SecurityBestPractices: arn:aws:inspector:us-west-1:166987590008:rulespackage/0-byoQRFYm
+    us-west-2:
+      CommonVulnerabilitiesExposures: arn:aws:inspector:us-west-2:758058086616:rulespackage/0-9hgA516p
+      CISOperatingSystemBenchmarks: arn:aws:inspector:us-west-2:758058086616:rulespackage/0-H5hpSawc
+      NetworkReachability: arn:aws:inspector:us-west-2:758058086616:rulespackage/0-rD1z6dpl
+      SecurityBestPractices: arn:aws:inspector:us-west-2:758058086616:rulespackage/0-JJOtZiqQ
+Resources:
+  InspectorServiceLinkedRole:
+    Type: AWS::IAM::ServiceLinkedRole
+    OrganizationBinding: !Ref InspectorBinding
+    Properties:
+      AWSServiceName: inspector.amazonaws.com
+      Description: Service-Linked-Role for Amazon Inspector
+  AssessmentTarget:
+    Type: AWS::Inspector::AssessmentTarget
```

Comment:
I think we would want to use InspectorV2, as it runs through the SSM agent and can examine ECR containers, doesn't run on a cron schedule, and doesn't accept targets or rule packages likes this (which is V1): https://docs.aws.amazon.com/inspector/latest/user/getting_started_tutorial.html